blockchain/fullblocks_test.go

```diff
@@ -714,3 +714,110 @@ func TestPhononActivation(t *testing.T) {
 		}
 	}
 }
+
+// TestCosmicInflationActivation tests that 64 bit integers and native introspection
+// opcodes activate correctly.
+func TestCosmicInflationActivation(t *testing.T) {
+	tests, err := fullblocktests.GeneratePhononBlocks()
```

Comment:
Is this supposed to be `GenerateCosmicInflationBlocks()` ? I tried replacing it but it fails out on this one:

```
   fullblocks_test.go:754: block "b1" (hash 587dad68d2dea0db396a69feb8f8645fe714b1bb5f11a126b8e6d0bee4f19d5f, height 102) should have been accepted: serialized transaction is too small - got 86, min 100
```

Reply:
ah yeah